python/oneflow/test/modules/test_initializer.py

```diff
@@ -155,7 +157,14 @@ def __call__(self, test_case, tensor):
             min=-0.10825317547305482, max=0.10825317547305482, mean=0.0, std=0.0625
         ),
     },
-    # TODO: test more initializer
+    # oneflow.nn.init.eye_
+    {
+        "func": flow.nn.init.eye_,
+        "params":{},
+        "checker": DataChecker(
+            function=np.eye,param=256
+        ),
+    },
```

Comment:
```suggestion
    # oneflow.nn.init.eye_
    {
        "func": flow.nn.init.eye_,
        "params": {},
        "checker": DataChecker(lambda_func=lambda size: np.eye(*size)),
    },
```

上面把 lambda 改成 lambda_func，避免和 python 的 lambda 命名冲突

Reply:
done